.github/workflows/sync-docs-full.yml

```
@@ -18,27 +18,42 @@ jobs:
-            done < mdx_files.txt | jq -s '.'
-          ) \
-          --slurpfile removed <(cat mdx_files.txt | jq -R . | jq -s .) \
-          --arg repo "$GITHUB_REPOSITORY" \
-          '{
-            repo: $repo,
-            changed: ($paths | .[0] // []),
-            removed: ($removed | .[0] // [])
-          }' > payload.json
+          rm -f changed_files.json
+          echo "[]" > changed_files.json
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)